package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "node make.js build",
     "test": "node make.js test",
+    "testNoBuild": "node make.js testNoBuild",
```

Comment:
Does the yaml script need to be updated?

Reply:
I think I did this above